development/components/form/form-theme/form-pages.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Form pages
+weight: 40
+---
+
+# Form pages
+
+This section states when a page that contains a Symfony form has been migrated to Symfony, and when the form has been simplified to rely on the PrestaShop UI Kit Form theme.
+
+You can find the detailed status of the project on the [GitHub tracking issue][form-theme-simplification-project].
+
+[form-theme-simplification-project]: https://github.com/PrestaShop/PrestaShop/issues/16482
+
+
+Page                      | Migrated in version       | Form simplified in version
+--------------------------|---------------------------|-------------------------------
+Advanced Parameters > Performance                                     | 1.7.3 | 1.7.8
+Advanced Parameters > Administration                                  | 1.7.4 | 1.7.8
+Advanced Parameters > Import                                          | 1.7.4 | 1.7.8
+Advanced Parameters > Logs                                            | 1.7.5 | 1.7.8
+Advanced Parameters > Database > DB Backup                            | 1.7.5 | 1.7.8
+Advanced Parameters > Database > SQL Manager > Add new SQL query      | 1.7.5 | 1.7.8
+Advanced Parameters > Webservice                                      | 1.7.5 | 1.7.8
+Advanced Parameters > Webservice > Add new webservice key             | 1.7.6 | 1.7.8
+Advanced Parameters > Email                                           | 1.7.5 | 1.7.8
+Advanced Parameters > Database > SQL Manager                          | 1.7.5 | 1.7.8
+Advanced Parameters > Team > Employees                                | 1.7.6 | 8.0.0
+Advanced Parameters > Team > Employees > Add new employee             | 1.7.6 | 8.0.0
+Advanced Parameters > Team > Employees > Profiles > Add new profile   | 1.7.6 | 8.0.0
```

Comment:
It makes no sense adding information for v8 in the 1.7 documentation, don't you think?

Reply:
Then "Not yet simplified" ?

Reply:
You can either leave it blank, or put something like "N/A"